projects/plugins/jetpack/modules/likes.php

```diff
@@ -448,6 +448,9 @@ function_exists( '\Activitypub\is_activitypub_request' )
 
 		$title = esc_html__( 'Like or Reblog', 'jetpack' );
 
+		/** This filter is documented in modules/likes/jetpack-likes-master-iframe.php */
+		$src = apply_filters( 'wpl_iframe_src', $src );
```

Comment:
Seems to me that only the `likes/master.html` iframe needs the `slim=1` parameter. On the `likes/index.html` iframe the parameter has no effect, because the document is just a small and passive placeholder. All the work is done by the master iframe: it loads the likes from REST, decides whether the UI will be slim or not and then replaces the DOM markup in all the managed iframes.

Reply:
I had been shuffling some stuff in my testing and thought it was required there as well -- or it wasn't propagating properly without it.  I could have been mistaken though.